Handling fee in confirm order mailer

dfreerksen opened the pull request:
# Summary

Note about mailers in README. Wiki pages have been created [here](https://github.com/groundctrl/spree_handling_fees/wiki/HTML-Order-Mailer) and [here](https://github.com/groundctrl/spree_handling_fees/wiki/Text-Order-Mailer)

# What's New

- Note about mailers in README

/cc @comeara

comeara commented:
We discussed the issues and think the email template override is the only viable solution for now.

It's debatable whether the template override belongs in this engine or if it should be documented here and left to the Rails app to define the template.

dfreerksen commented:
I've updated the PR. I removed the view overrides I had. Instead I made a new section in the README explaining the issue. I made a couple wiki pages to explain what needs to happen in the Rails app.

comeara commented:
👍
